addons/account_fleet/models/fleet_vehicle_log_services.py

```diff
@@ -0,0 +1,36 @@
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+class FleetVehicleLogServices(models.Model):
+    _inherit = 'fleet.vehicle.log.services'
+
+    account_move_line = fields.Many2one('account.move.line')
+    account_move_state = fields.Selection(related="account_move_line.parent_state", store=True)
+    account_move = fields.Many2one('account.move', related="account_move_line.move_id", store=True)
+    amount = fields.Monetary('Cost', compute="_compute_amount", readonly=False, store=True, tracking=True)
+
+    @api.depends('account_move_state')
+    def _compute_amount(self):
+        for log_service in self:
+            if log_service.account_move_line:
+                log_service.amount = log_service.account_move_line.price_subtotal
+
+    def action_open_account_move(self):
+        self.ensure_one()
+        return {
+            'type': 'ir.actions.act_window',
+            'view_mode': 'form',
+            'res_model': 'account.move',
+            'target': 'current',
+            'name': _('Bills'),
+            'res_id': self.account_move.id,
+        }
+
+    def unlink(self):
+        for log_service in self:
+            if log_service.account_move_line:
+                raise UserError(
+                _('You cannot delete log services records because one or more of them were bill created.')
+            )
+        return super().unlink()
```

Comment:
```suggestion
        if any(log_service.account_move_line for log_service in self):
            raise UserError(_('You cannot delete log services records because one or more of them were bill created.'))
        return super().unlink()
```